packages/gatsby/src/redux/nodes.ts

```diff
@@ -260,11 +257,13 @@ exports.ensureIndexByTypedChain = ensureIndexByTypedChain
  *   This object lives in query/query-runner.js and is passed down runQuery
  * @returns {Array<Node> | undefined}
  */
-const getNodesByTypedChain = (
-  chain,
-  value,
-  nodeTypeNames,
-  typedKeyValueIndexes
+export const getNodesByTypedChain = (
+  chain: string[],
+  value: boolean | number | string,
+  nodeTypeNames: string[],
+  typedKeyValueIndexes:
+    | undefined
```

Comment:
I _think_ this object cannot be `undefined` at this point. Loki skips this entirely and non-loki should have set the cache by now.